File: IDPA/Programming Labs 101/INCLASS/Java/Classes/src/Cat.java

public class Cat
{
	//Constructors
	public Cat() //default
	{
		name = "GenericCat";
		age = 2;
	}
	
	public Cat(String name, int age) //dynamic
	{
		this.name = name;
		this.age = age;
	}
	
	//Functions
	public void meow()
	{
		System.out.println("Meow!");
	}
	
	//Variables
	public String name;
	public int age;
}

File: IDPA/Programming Labs 101/Javascript/Classes and Functions/cat.js
var meowCount;

//Dynamic Constructor
function Cat(name, age)
{	
	this.name = name;
	this.age = age;
	
	this.meow = function(numOfTimes)
	{
		var x = "";
		
		for(var i = 0; i < numOfTimes; i++)
		{
			x += "Meow!" + "<br>";
			document.getElementById("meow").innerHTML = x;
		}
	}
}

function Meow()
{
	meowCount = parseInt(prompt("How many times will the cat meow?", "Please enter here"));
	ginger.meow(meowCount);
}

var ginger = new Cat("Ginger", 10);

